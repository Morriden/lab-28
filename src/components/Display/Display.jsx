import React from 'react';
import PropTypes from 'prop-types';

const Display = ({ responseInfo }) => (
    <section>
        <div>
            {responseInfo}
        </div>
    </section>
)

Display.propTypes = {
    responseInfo: PropTypes.string.isRequired
}

export default Display;
