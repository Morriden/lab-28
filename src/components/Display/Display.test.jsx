import React from 'react';
import { shallow } from 'enzyme';
import Display from './Display';

describe('Form component', () => {
  it('should render the display section', () => {
    const wrapper = shallow(
    <Display 
        responseInfo="I'm a JSON File"
    />);
    expect(wrapper).toMatchSnapshot();
  });
});