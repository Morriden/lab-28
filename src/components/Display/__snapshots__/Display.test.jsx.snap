// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form component should render the display section 1`] = `
<section>
  <div>
    I'm a JSON File
  </div>
</section>
`;
